src/broker/amqp.rs

```diff
@@ -192,14 +194,22 @@ impl Broker for AMQPBroker {
             .await
             .basic_consume(
                 queue.name().as_str(),
-                "",
```

Comment:
If you want to avoid generating an uuid, you can let `""` and fetch the one generated by the server using https://docs.rs/lapin/1.6.5/lapin/struct.Consumer.html#method.tag